examples/pagerank/cmake/apache-arrow.cmake

```diff
@@ -0,0 +1,92 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Ditto. We can reuse the apache-arrow.cmake of GraphAr.

Reply:
I prefer remove arrow from our example